test-files/services-system/QuietReporter.js

```diff
@@ -0,0 +1,23 @@
+module.exports = QuietReporter;
+
+function QuietReporter(runner) {
+  var passes = 0;
+  var failures = 0;
+
+  runner.on('pass', function(test){
+    passes++;
+  });
+
+  runner.on('fail', function(test, err){
+    failures++;
+    console.log('fail: %s -- error: %s', test.fullTitle(), err.message);
+  });
+
+  runner.on('end', function(){
+    if (failures > 0)
+    {
+        console.log('end: %d/%d', passes, passes + failures);
+    }
+    process.exit(failures);
+  });
+}
```

Comment:
There should be no such path: test-files/services-system.  This is redundant with hoot-tests/services-system.child.  It once was in this location but later was removed, so must be there now as the result of a bad merge.